routes/admin.rb

```diff
@@ -623,6 +627,10 @@
     udo_template.destroy
   end
   @udos_templates = UserDefinedObjectTemplates.all
+
+  @admin = true if is_administrator?
```

Comment:
these should be moved to the top of the route if they are meant to enforce access control. I think this is taken care of at the top of the route, but I am good with a secondary check.

Reply:
Never mind =). I see it's for the template.